Makefile

```diff
@@ -11,18 +11,19 @@ compile:
 	./gradlew build
 
 # do a dry release (like a local deploy)
+# remember to add the -SNAPSHOT suffix to the version
```

Comment:
remembering is hard, can we check for that before running this task? Or not really a big deal if we add a non SNAPSHOT version to local maven?

Reply:
this is already part of our release manual and ideally should be automatized by Craft which is not the focus of this PR